package.json

```diff
@@ -7,15 +7,21 @@
   "bin": {
     "bunyan": "./bin/bunyan"
   },
-
```

Comment:
NPM seems to have reformtted a lot of the package.json with `npm i -D webpack`, should I revert these changes.